cypress/support/step_definitions/annual-review/scheduler/the_unpublish_reminder_email_is_not_sent.js

```diff
@@ -0,0 +1,49 @@
+Then("the unpublish reminder email is not sent", async () => {
+  cy.task("db", {
+    operation: "list.findFirst",
+    variables: {
+      where: {
+        reference: "SMOKE",
+      },
+    },
+  }).then((list) => {
+    cy.task("db", {
+      operation: "audit.findFirst",
+      variables: {
+        where: {
+          type: "list",
+          auditEvent: "ANNUAL_REVIEW",
+          jsonData: {
+            path: ["itemId"],
+            equals: list.id,
+          },
+        },
+      },
+    }).then((audit) => {
```

Comment:
@RichardBray one of your tests was failing since it was looking for list.jsonData.currentAnnualReview.reference but by that point the list would've already been reset (the annualReviewObject deleted). I've added this step which first gets the list id, then look for audit events that match to get the ref, then finally check if there are listItem Events that match